src/features/movies/MovieList/Container/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    max-width: 1368px;
+    max-height: 1391px;
+    margin-left: 276px;
+    margin-top: 150px;
+`;
+
+export const Container = styled.div`
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 24px;
+    
+    @media (max-width: ${({ theme }) => theme.breakpoints.tablet}){
+        display: grid;
```

Comment:
W `@media` nie musisz ponownie pisać `display: grid`. Masz to zdeklarowane wyżej. W `@media` dodajesz tylko te właściwości, które chcesz zmienić lub dodać.

Reply:
Już usunąłem zbędne linijki kodu